src/answers-umd.js

```diff
@@ -79,6 +80,11 @@ class Answers {
      */
     this.renderer = new Renderers.Handlebars();
 
+    /**
+     * A reference of AlertifyJS library for browser dialogs and notifications.
+     */
+    this.alertify = alertify;
```

Comment:
Based on the other PR, it doesn't look like the theme needs access to an alert function, so I don't think we need to add it to ANSWERS yet. Tom likes to keep the functionality on ANSWERS to a minimum